app/graphql/types/hmis_schema/mutation_type.rb

```diff
@@ -1,3 +1,9 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
 # frozen_string_literal: true
```

Comment:
I believe this only works if its on the first line.  We may need to adjust our copyright cleanup script to handle that if we're going to use it in this form.

Reply:
I just tested this, and it seems that this magic comment works after the copyright:
```
➜  Tmp cat magic.rb                                              <aws:openpath>
###
# Copyright 2016 - 2022 Green River Data Analysis, LLC
#
# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
###

# frozen_string_literal: true

puts 'foo'.frozen?
➜  Tmp ruby magic.rb 
true
```